Simplenote/UntouchableView.swift

```diff
@@ -0,0 +1,16 @@
+import UIKit
+
+class UntouchableView: UIView {
+    override func point(inside point: CGPoint, with event: UIEvent?) -> Bool {
```

Comment:
I **suspect** you may wanna override hitTest instead maybe?

Reply:
As discussed in slack, I checked out PassthruView and that works to do what we were looking for here.  Have removed the UntouchableView file.